browser/extensions/BUILD.gn

```diff
@@ -146,6 +147,13 @@ source_set("extensions") {
   if (enable_brave_wayback_machine) {
     deps += [ "//brave/components/brave_wayback_machine" ]
   }
+
+  if (enable_ai_chat) {
+    sources += [
+      "api/leo_assistant_api.cc",
```

Comment:
Actually - see @petemill's [comment](https://github.com/brave/brave-core/pull/19212#pullrequestreview-1525175816) - it's much more sensible than setting up a mojom API :laughing: Sorry for the distraction

Reply:
Moved all things to the `BraveLeoAssistantHandler` instead of extension API, It uses existing `content/common/web_ui.mojom` interface, are you sure I have to define something new?

Reply:
@vadimstruts you don't. The options for passing data back and forth from browser to webui are either generic MessageHandler or types mojom. We're moving settings handlers to mojom, but many are still MessageHandlers I believe.

Reply:
Sorry for the confusion @vadimstruts!